src/styles/blocks/card.scss

```diff
@@ -0,0 +1,111 @@
+/* #region card */
+.card {
+  padding-top: 32px;
+  padding-inline: 16px;
+  padding-bottom: 16px;
```

Comment:
It can be simplified -> padding

Reply:
I can`t centering card__img. Bacause of this my tests are failed. When i add margin or padding on website page everything looks good but tests don`t see this cganges.